Makefile

```diff
@@ -66,6 +66,11 @@ prepare-release: cross
 test:
 	go test -race $(PKGS)
 
+# Run e2e tests
+.PHONY: test-e2e
+test-e2e:
+	go test -v github.com/redhat-developer/ocdev/tests/e2e
```

Comment:
do we really need to run it in verbose mode?

Reply:
Sure. WIll disable. Thought it would be useful to debug.